src/app/functions/universal/guidedExperienceBreaches.test.ts

```diff
@@ -35,7 +35,20 @@ it("getGuidedExperienceBreaches: return guided experience", () => {
     name: "",
     title: "",
     emailsAffected: ["test@mozilla.com"],
-    dataClassesEffected: [],
+    dataClassesEffected: [
+      { [BreachDataTypes.PIN]: 1 },
```

Comment:
Related to my other comment, maybe one of these should have multiple affected classes on the same data type? Or is this _meant_ to only hold a single data class? (In which case: ouch, I really wish it was a tuple instead...)

Reply:
No, it can hold more data classes. :) Added a second one in [cf21d5d](https://github.com/mozilla/blurts-server/pull/3791/commits/cf21d5d978fb7d9ad02217b6dd6feccc2089c4cb).